### PR TITLE
Add Steppers component

### LINKED ISSUE #19: Stepper: Create stepper component
[Figma Design](https://www.figma.com/file/msxmKajRPLYiDDmsP60Ndh/Design-System-v1.1?type=design&node-id=6576-51106&t=ttVz3vzOxnYJ4iYn-4)

should support:

- [x] vertical / horizontal orientation
- [x] state (active / inactive / error)

### PR DESCRIPTION
**Pull request by tewshi**
Closes #19

### CODE REVIEW

**Comment by lukicenturi** [comment]
Is there anyway to show the current step like this?
image

I can't find it in the example

File: src/components/steppers/Stepper.vue [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+      @apply text-rui-warning;
+    }
+
+    &.info {
+      @apply text-rui-info;
+    }
+
+    &.success {
+      @apply text-rui-success;
+    }
```
Comment (lukicenturi) [inline src/components/steppers/Stepper.vue:102]: I create the variable $context-colors in my latest PR, #39, so you can loop this in scss.

**Comment by tewshi** [comment]
> Is there anyway to show the current step like this? image
> 
> I can't find it in the example

yeah, you're right.

it was labelled as custom, so I made a slot for custom icon, but I'll add a wrapper and use the custom icons as we have here.

File: src/components/steppers/Stepper.vue (hunk 1 above)
Comment (tewshi) [reply src/components/steppers/Stepper.vue:102]: maybe I can work on this in my next PR, when yours is merged... what do you think?

Comment (tewshi) [reply src/components/steppers/Stepper.vue:102]: oh, it is not a mixin, I'll add it

Comment (lukicenturi) [reply src/components/steppers/Stepper.vue:102]: It's fine too, if you want to add it after my PR

Comment (tewshi) [reply src/components/steppers/Stepper.vue]: I've actually added it. thanks.

**Comment by lukicenturi** [comment]
@tewshi I think for the custom one, you need to force the color to use `rui-light-primary` (force to use light mode primary version, in both light and dark mode) referring to the Figma.

Figma: 
image

Yours:
image

**Comment by lukicenturi** [comment]
for the stepper with primary color, I think you need to set the color to black (white in dark mode), instead of primary (?), so then we can remove the `customTitleColor`.

Yours:
image

Figma:
image

**Review by lukicenturi — APPROVED** [review]
LGTM!

**Review by kelsos — APPROVED** [review]
lgtm

**Pull request merged**